src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -6608,6 +6608,17 @@ public override IOperation VisitRangeOperation(IRangeOperation operation, int? a
             return new RangeOperation(operation.IsLifted, operation.IsImplicit, semanticModel: null, operation.Syntax, operation.Type, visitedLeftOperand, visitedRightOperand, operation.Method);
         }
 
+        public override IOperation VisitSuppressNullableWarningOperation(ISuppressNullableWarningOperation operation, int? argument)
+        {
+            IOperation visitedExpression = null;
+            if (!(operation.Expression is null))
+            {
+                visitedExpression = Visit(operation.Expression);
+            }
+
+            return new SuppressNullableWarningOperation(operation.IsImplicit, semanticModel: null, operation.Syntax, operation.Type, visitedExpression);
```

Comment:
>operation.IsImplicit [](start = 56, length = 20)

```IsImplicit(operation)```

Reply:
It looks like two methods above have the same problem. Please fix them too.

---
In reply to: [244069749](https://github.com/dotnet/roslyn/pull/32019#discussion_r244069749) [](ancestors = 244069749)